pom.xml

```diff
@@ -168,8 +177,8 @@
 			<id>openmrs-1.9</id>
 			
 			<properties>
-				<openMRSVersion>1.9.9</openMRSVersion>
-				<openMRSMinorVersion>1.9</openMRSMinorVersion>
```

Comment:
Does it successfully compile on Java 8 if you change this back to 1.9.9?

Reply:
No. still get the same error